testing/matrix-sdk-test/src/test_json/sync.rs

```diff
@@ -1357,6 +1360,30 @@ pub static MIXED_SYNC: Lazy<JsonValue> = Lazy::new(|| {
                   }
                 }
             },
+            "knock": {
+                *MIXED_KNOCKED_ROOM_ID: {
+                  "knock_state": {
+                    "events": [
+                      {
+                        "sender": "@alice:example.com",
+                        "type": "m.room.name",
+                        "state_key": "",
+                        "content": {
+                          "name": "My Room Name"
+                        }
+                      },
+                      {
+                        "sender": "@alice:example.com",
+                        "type": "m.room.member",
+                        "state_key": "@bob:example.com",
```

Comment:
Can someone knock on behalf of somebody else? :thinking: Should the sender/state_key be the same?

Reply:
Ooops. Copy&paste issue 😅 .